wordcloud/wordcloud.py

```diff
@@ -310,6 +310,20 @@ def __init__(self, font_path=None, width=400, height=200, margin=2,
                           " it had no effect. Look into relative_scaling.",
                           DeprecationWarning)
         self.normalize_plurals = normalize_plurals
+        
+    @property
```

Comment:
why not just set it in the constructor based on ``mask``? In case someone changes ``mask`` later? Can you please also add a test?

Btw, it might also be helpful to set ``height`` and ``width`` to ``None`` and if no mask is provided set them to the default value, and if a value is provided and a mask is also provided raise an error. that should make debugging easier maybe?